### PR TITLE
Clean up E0724 explanation

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0724.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-`#[ffi_returns_twice]` was used on non-foreign function.
+`#[ffi_returns_twice]` was used on something else than a foreign function
```
Comment (jyn514) [inline src/librustc_error_codes/error_codes/E0724.md:1]:
```suggestion
`#[ffi_returns_twice]` was used on something other than a foreign function
```
Is this more clear than before? What happens if you try and put this over a struct or something, is it the same error?

**Comment by GuillaumeGomez** [comment]
Updated!

**Pull request merged**